mungegithub/submit-queue/deployment/nginx-redirect/nginx-deploy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: nginx-https-redirect
+spec:
+  replicas: 1 
+  template:
+    metadata:
+      labels:
+        app: nginx-https-redirect
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+        ports:
+        - containerPort: 80 
+        volumeMounts:
+        - name: conf
+          mountPath: /etc/nginx
+      volumes:
+      - name: conf
+        configMap:
+          name: nginx-https-redirect
```

Comment:
It contains the contents of the `nginx.conf` file in this PR. Would you like me to include the config map as well?

I created it with:

```
kubectl create cm nginx-https-redirect --from-file=nginx.conf=nginx.conf
```

(I could add a README too, I suppose)